scripts/dev_setup.sh

```diff
@@ -428,6 +436,28 @@ function install_cvc4 {
   rm -rf "$TMPFILE"
 }
 
+function install_codegen_deps {
+    if [[ "$PACKAGE_MANAGER" != "apt-get" ]]; then
+      # Not supported.
+      return
+    fi
+    echo "deb http://http.us.debian.org/debian/ buster-backports main" | "${PRE_COMMAND[@]}" tee -a /etc/apt/sources.list
+    echo "deb-src http://http.us.debian.org/debian/ buster-backports main" | "${PRE_COMMAND[@]}" tee -a /etc/apt/sources.list
+    "${PRE_COMMAND[@]}" apt-get update
+    "${PRE_COMMAND[@]}" apt-get upgrade -y
+    "${PRE_COMMAND[@]}" apt-get install -y apt-transport-https python3-all-dev python3-pip clang llvm default-jdk golang-1.14-go/buster-backports
```

Comment:
this script will be run in multiple containers and should be tested in them, there is a package install script you should be using, that will attempt to use the correct package manager.

Reply:
including mac.

Reply:
IMHO we should focus on having the SDK codegen covered by CI in order to prevent regressions.

Reply:
I refactored the code to look more like the rest of the file.